cfgov/unprocessed/js/routes/common.js

```diff
@@ -19,7 +19,6 @@ require( '../modules/scroll-on-history-collapse' ).init();
 require( '../modules/clear-form-buttons' ).init();
 require( '../modules/UStreamPlayer' ).init( '.video-player__ustream' );
```

Comment:
you are correct but the event page doesn't have an `page.media` object. @kave, is there a way to add it to the Event page?